api/nakadi-event-bus-api.yaml

```diff
@@ -59,21 +347,42 @@ paths:
             items:
               $ref: '#/definitions/Topic'
         '401':
-          description: User is not authenticated
+          description: Client is not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+        '500':
+          description: Server error
           schema:
             $ref: '#/definitions/Problem'
+
         '503':
           description: Not available
           schema:
             $ref: '#/definitions/Problem'
+
   '/topics/{topic}/events':
     get:
       tags:
-        - low-level-api
-      summary: |
+        - topic-api
+        - draft
+      description: |
         Starts a stream delivery for the specified partitions of the given topic.
-        The tracking of the current position in the partitions and of which
-        partitions is being read is in the responsibility of the client.
+
+        The event stream is formated as a sequence of `EventStreamBatch`es separated by `\n`. Each
```

Comment:
I just want to clarify how current streaming from multiple partitions works (to what we agreed before implementing streaming from multiple partitions)
For example for batch_limit 5 you will get something like (this displays three batches)
```json
{"cursor":{"partition":"0","offset":"4555"}, "events":[{"thisIs":"dummyEvent"}, 
                                                       {"thisIs":"dummyEvent"}, 
                                                       {"thisIs":"dummyEvent"}]}\n
{"cursor":{"partition":"1","offset":"2365"}, "events":[{"thisIs":"dummyEvent"}]}\n
{"cursor":{"partition":"2","offset":"6544"}, "events":[{"thisIs":"dummyEvent"}]}\n
\n
{"cursor":{"partition":"2","offset":"6546"}, "events":[{"thisIs":"dummyEvent"}, 
                                                       {"thisIs":"dummyEvent"}]}\n
{"cursor":{"partition":"3","offset":"9875"}, "events":[{"thisIs":"dummyEvent"}, 
                                                       {"thisIs":"dummyEvent"}, 
                                                       {"thisIs":"dummyEvent"}]}\n
\n
{"cursor":{"partition":"0","offset":"4556"}, "events":[{"thisIs":"dummyEvent"}]}\n
{"cursor":{"partition":"1","offset":"2366"}, "events":[{"thisIs":"dummyEvent"}]}\n
{"cursor":{"partition":"2","offset":"6547"}, "events":[{"thisIs":"dummyEvent"}]}\n
{"cursor":{"partition":"3","offset":"9877"}, "events":[{"thisIs":"dummyEvent"}, 
                                                       {"thisIs":"dummyEvent"}]}\n
\n
...
```